README.md

```diff
@@ -1,6 +1,8 @@
-# Scoobie
+# 🧶 Scoobie
 
-[![npm package](https://img.shields.io/badge/npm-scoobie-cc3534.svg)](https://www.npmjs.com/package/scoobie)
+![GitHub Release](https://github.com/seek-oss/scoobie/workflows/Release/badge.svg?branch=master)
+![GitHub Validate](https://github.com/seek-oss/scoobie/workflows/Validate/badge.svg?branch=master)
+[![npm package](https://img.shields.io/npm/v/scoobie)](https://www.npmjs.com/package/scoobie)
 [![Slack channel](https://img.shields.io/badge/slack-%23indirect--apply-3f0f3f.svg)](https://slack.com/app_redirect?team=T02P37LGR&channel=indirect-apply)
```

Comment:
I wonder if we should kill the Slack/Upkeep badges. They weren't exactly mission critical even in SEEK-Jobs.